lib/text/mention_text_editing_controller.dart

```diff
@@ -1,121 +1,76 @@
+// ignore_for_file: constant_identifier_names
 import 'dart:async';
 
 import 'package:diff_match_patch/diff_match_patch.dart';
 import 'package:flutter/material.dart';
 
-// ignore_for_file: constant_identifier_names
-
-// Mention object that store the id, display name and avatarurl of the mention
-// You can inherit from this to add your own custom data, should you need to
-
-// Keep in copy with diff.dart from diff_match_patch package
```

Comment:
Keep this comment "Keep in copy with diff.dart from diff_match_patch package"

Reply:
reolled back